docs/sources/flow/tasks/configure/configure-kubernetes.md

```diff
@@ -61,5 +81,9 @@ configMapGenerator:
       disableNameSuffixHash: true
 ```
 
+## Configure {{< param "PRODUCT_NAME" >}}
+
+TODO: How to change the river config that is used by the Helm chart.
```

Comment:
This feels tutorialish with the broadness of the topic. My thought is remove/move this topic or make it more pointed? IE configure HPA for agent metrics collection. At the end of this it feels like they have the agent running in kubernetes doing very little.

Reply:
Ok, I think I can remove it for now. Other tasks may come in the future like you mention.